fab

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+ENV=.deployment_virtualenv
+# version of fusionbox-fabric-helpers to run
+HELPERS_COMMIT=f854d6a00
+HELPERS_COMMIT_FILE=$ENV/HELPERS_COMMIT
+
+if [ ! -d $ENV ]
+then
+  virtualenv -p `which python2` $ENV
+  touch $HELPERS_COMMIT_FILE
+fi
+source $ENV/bin/activate
+if [ "$HELPERS_COMMIT" != "`cat $HELPERS_COMMIT_FILE`" ]
```

Comment:
Visually backticks can be confused with single quotes, also it makes it hard to parse for human. Expensions in strings is done by `$...` with the exception of backticks. When seeing a `$` sign people go "oh... this is gonna get expended", but with backticks (I mean it was the case for me) people go "what did he mean here? wait.. there are backticks... I see"